guestlink.php

```diff
@@ -59,6 +59,9 @@
     if ($guestpass && $bigbluebuttonbn->guestpass != $guestpass) {
         $guestpasserrormessage = true;
     }
+    if (isset($guestname) && !$guestname) {
```

Comment:
What is this actually checking for?

Reply:
If guestname is not provided, it won't show an error saying "Please provide a name"

If guestname was submitted but was empty (somehow), it will show the error. 

This was part of a block which seemed to have the error but it was never reacahable.